src/user.js

```diff
@@ -0,0 +1,15 @@
+import users from './users.json';
+
+export function seedUserData(){
+  var userModules = users.students[0].tracked_modules;
+  return hoodie.store.add({
+    "userModules": userModules
+  })
+  .then(() => {
+    console.log('your initial data has been given to hoodie');
+    return userModules;
```

Comment:
`userModules` is undefined here. If you would like to return some data here, it should look like this: `return Promise.resolve(userModules);`.